### DISCUSSION TITLE
Copilop doesn't work on some of my repositories

### DISCUSSION THREAD

**Discussion by Avivhdr** [community/community#49233] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

**Problem**
The Github copilot extension doesn't generate suggestions on all of my repositories.
Please take a look at the below GIF.

**Attempts to solve the issue:**

- [x] Restarting VS code
- [x] Signing out of Github Copilop VS code extension and logging back in.

![Kapture 2023-03-05 at 15 54 58](https://user-images.githubusercontent.com/23265867/222968172-0af4bd56-490f-4cf4-bf82-e280002a13f6.gif)

**Comment by iamcjwilliams** [comment, marked as answer]
@Avivhdr please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot
